gui/map.cc

```diff
@@ -69,28 +76,57 @@ Map::Map(QWidget *parent,
   manager_ = new QNetworkAccessManager(this);
   connect(this,SIGNAL(loadFinished(bool)),
 	  this,SLOT(loadFinishedX(bool)));
-  this->logTimeX("Start map constuctor");
+  this->logTime("Start map constuctor");
```

Comment:
Is /this/ ever needed in this file?

Reply:
Are you asking about using "this->" syntax, or are you asking about the logTime(X) functions?  The logTime functions track various timings and report to the gui console if you enable debug in the options dialog.  I think this used to be more important with all the encoded polylines, but they are gone with v3 except for driving directions.